2021/01/21 10:30:16 : Error Connecting:HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x109b57a90>: Failed to establish a new connection: [Errno 61] Connection refused')) : False
2021/01/21 10:34:34 : Error Connecting:HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107215c50>: Failed to establish a new connection: [Errno 61] Connection refused')) : False
2021/01/21 10:34:59 : Error Connecting:HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x107cfbb50>: Failed to establish a new connection: [Errno 61] Connection refused')) : False
